app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-core.js

```diff
@@ -78,12 +78,11 @@ window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {};
       },
 
       hrefIsRelative: function (href) {
-        // Checks that a link is relative, but is not a protocol relative url
-        return href[0] === '/' && href[1] !== '/'
-      },
+        // Checks that a href is relative by the lack of http:, https:// or // at the start of the href.
+        var domainRegex = /^(http:||https:)?(\/\/)([^\/]*)/ // eslint-disable-line no-useless-escape
+        var domain = domainRegex.exec(href)
 
-      hrefIsAnchor: function (href) {
-        return href[0] === '#'
+        return !domain
```

Comment:
I had this return statement a bit more literal, but standardx keeps simplifying it.